app/connectors/GetCalculationListConnector.scala

```diff
@@ -17,23 +17,23 @@
 package connectors
 
 import config.AppConfig
-import connectors.httpParsers.GetCalculationListHttpParser._
+import connectors.httpParsers.GetCalculationListHttpParser.{GetCalculationListHttpReads, GetCalculationListResponse}
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
 
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class GetCalculationListConnector @Inject()(http: HttpClient, val appConfig: AppConfig)(implicit ec: ExecutionContext) extends DesConnector {
+class GetCalculationListConnector @Inject()(httpClient: HttpClient,
+                                            val appConfig: AppConfig)
+                                           (implicit ec: ExecutionContext) extends IFConnector {
 
-  def calcList(nino: String, taxYear: Option[String])(implicit hc: HeaderCarrier): Future[GetCalculationListResponse] = {
+  def getCalculationList(nino: String)(implicit hc: HeaderCarrier): Future[GetCalculationListResponse] = {
+    val getCalculationListUrl: String = appConfig.ifBaseUrl + s"/income-tax/view/calculations/liability/23-24/$nino"
```

Comment:
you need to pass TaxYear instead of fix it to 23-24, pls I think actual format is YYYY-YY

Reply:
In the schema for the 1896 API, it says the tax year parameter can only be the values listed in the enum. 

The enum only lists `23-24`
```
  parameters:    
    taxYearParam:  
      in: path
      name: taxYear
      description: The tax year to which the entity relates.  23-24 for the 23/24 tax year.
      required: true
      schema:
        type: string
        enum: ['23-24']
```